spark/components/pagination/vanilla/pagination.stories.js

```diff
@@ -1,3 +1,217 @@
+import { useEffect } from '@storybook/client-api';
+import { pagination } from './pagination';
+
 export default {
   title: 'Components|Pagination',
 };
+
+export const defaultStory = () => {
+  useEffect(() => {
+    pagination();
+  }, []);
+
+  return `
+  <nav
+    aria-label="Pagination Navigation"
+    data-sprk-pagination="default"
+    data-id="pagination-1"
+  >
+    <ul
+      class="sprk-c-Pagination sprk-o-HorizontalList sprk-o-HorizontalList--spacing-medium"
+    >
+      <li>
+        <a
+          class="sprk-c-Pagination__icon sprk-b-Link sprk-b-Link--disabled sprk-b-Link--plain"
+          href="#"
+          data-sprk-pagination="prev">
+          <span class="sprk-u-ScreenReaderText">Prev</span>
+          <svg
+            class="sprk-c-Icon sprk-c-Icon--stroke-current-color"
+            viewBox="0 0 64 64"
+          >
+            <use xlink:href="#chevron-left"></use>
+          </svg>
+        </a>
+      </li>
+
+      <li>
+        <a
+          class="sprk-c-Pagination__link sprk-c-Pagination__link--current sprk-b-Link"
+          href="#"
+          data-sprk-pagination="item"
+          aria-current="true">
+          1
+        </a>
+      </li>
+
+      <li>
+        <a
+          class="sprk-c-Pagination__link sprk-b-Link"
+          href="#"
+          data-sprk-pagination="item">
+          2
+        </a>
+      </li>
+
+      <li>
+        <a
+          class="sprk-c-Pagination__link sprk-b-Link"
+          href="#"
+          data-sprk-pagination="item">
+          3
+        </a>
+      </li>
+
+      <li>
+        <a
+          class="sprk-c-Pagination__icon sprk-b-Link sprk-b-Link--plain"
+          href="#"
+          data-sprk-pagination="next">
+          <span class="sprk-u-ScreenReaderText">Next</span>
+          <svg
+            class="sprk-c-Icon sprk-c-Icon--stroke-current-color"
+            viewBox="0 0 64 64"
+          >
+            <use xlink:href="#chevron-right"></use>
+          </svg>
+        </a>
+      </li>
+    </ul>
+  </nav>
+  `;
+};
+
+defaultStory.story = {
+  name: 'default',
+};
+
+export const long = () => {
```

Comment:
We want long capitalized in SB. You might have to add the long.story name setting if SB isn't capitalizing it.

Reply:
Storybook automatically capitalizes it if it isn't explicitly set. So we're good there. 
![image](https://user-images.githubusercontent.com/15703773/67104559-271ace00-f195-11e9-8898-979e56c82b23.png)